src/js/utils/quality-labels.js

```diff
@@ -69,11 +69,25 @@ define([
         return Math.floor(bandwidth / 1000);
     }
 
+    // Use an empty object as the context and populate it like a hash map
+    function hasRedundantLevels(levels) {
+        if (!_.isArray(levels)) {
+            return false;
+        }
+        return _.some(levels, function (level) {
+            var key = level.height || level.bandwidth;
+            var foundDuplicate = this[key];
+            this[key] = 1;
+            return foundDuplicate;
+        }, {});
+    }
```

Comment:
Also this looks like it could be used for audio tracks as well.

https://github.com/jwplayer/jwplayer-commercial/blob/v7.10/src/js/providers/shaka.js#L418